safekeeper/src/safekeeper.rs

```diff
@@ -639,7 +639,6 @@ where
 
             let mut state = self.state.clone();
             state.server.system_id = msg.system_id;
-            state.server.wal_seg_size = msg.wal_seg_size;
```

Comment:
why remove this line?

Reply:
Because we have a check about `wal_seg_size` on the line 625 above, it means that it's the same.

We don't support changing wal_seg_size in safekeepers, and AFAIK we cannot change it in the postgres as well.